# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GfxTexturesReporter.cpp
# Commit: 898ed02804fe
# Full Hash: 898ed02804fefc9786112f135b9621cc444e6860
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-23 15:52:15
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r e1b7abc99ae9 -r 898ed02804fe gfx/gl/GfxTexturesReporter.cpp
--- a/gfx/gl/GfxTexturesReporter.cpp	Wed May 22 12:44:04 2019 +0000
+++ b/gfx/gl/GfxTexturesReporter.cpp	Wed May 22 22:27:37 2019 +0000
@@ -8,7 +8,7 @@
 #include <sstream>
 #include "nsExceptionHandler.h"
 #include "GfxTexturesReporter.h"
-#include "gfxPrefs.h"
+#include "mozilla/StaticPrefs.h"
 
 using namespace mozilla;
 using namespace mozilla::gl;
@@ -59,7 +59,7 @@
         "GFX: Current texture usage greater than update amount.");
     sAmount -= amount;
 
-    if (gfxPrefs::GfxLoggingTextureUsageEnabled()) {
+    if (StaticPrefs::GfxLoggingTextureUsageEnabled()) {
       printf_stderr("Current texture usage: %s\n",
                     FormatBytes(sAmount).c_str());
     }
@@ -67,7 +67,7 @@
     sAmount += amount;
     if (sAmount > sPeakAmount) {
       sPeakAmount.exchange(sAmount);
-      if (gfxPrefs::GfxLoggingPeakTextureUsageEnabled()) {
+      if (StaticPrefs::GfxLoggingPeakTextureUsageEnabled()) {
         printf_stderr("Peak texture usage: %s\n",
                       FormatBytes(sPeakAmount).c_str());
       }